src/config/wmodules-azure.c

```diff
@@ -480,6 +468,22 @@ int wm_azure_storage_read(const OS_XML *xml, XML_NODE nodes, wm_azure_storage_t
                 OS_ClearNode(children);
             }
 
+        // To avoid the defects reported by scan-build in the issue: https://github.com/wazuh/wazuh/issues/11568
+        // It's necessary to check that nodes[i]->content is valid
```

Comment:
Remove this comments
```suggestion 
        // To avoid the defects reported by scan-build in the issue: https://github.com/wazuh/wazuh/issues/11568
        // It's necessary to check that nodes[i]->content is valid
```